templates/configmap.yaml

```diff
@@ -9,3 +9,4 @@ data:
   search_enabled: {{ .Values.config.search.enabled | quote }}
   search_url: '{{ .Values.config.search.url }}'
   workspaces_enabled: {{ .Values.config.workspaces.enabled | quote }}
+  analytics: {{ .Values.config.analytics | quote }}
```

Comment:
I think you're going to need to stringify the yaml map, when I run helm template I get the following output:
```
  analytics: "map[auth:map[ga_property: mixpanel_token:0c39f8410fd548bfa1976957d0248289] enabled:true platform:mixpanel]"
```